docs/Overview.md

```diff
@@ -73,6 +73,26 @@ The `statusReportingDelay` property allows clients to customize the reporting de
 |:--:|
 | _**Status reporting delay example**_ |
 
+### Receiver status transition counters
+
+All receiver specific domain statuses have an associated status transition counter property. These increment each time the associated status transitions to a less healthy state. Transitions to/from neutral states like `Inactive` or `NotUsed` are ignored.
+
+The intention is that these properties store historical negative trend transitions for each status.
+
+The list of all status transition counter properties is:
+
+* linkStatusTransitionCounter
+* connectionStatusTransitionCounter
+* externalSynchronizationStatusTransitionCounter
+* streamStatusTransitionCounter
+
+Devices MUST be able to reset ALL status transition counter properties in the following two ways:
+
+* When a receiver activation occurs
+* When a client invokes the `ResetStatusTransitionCounters` method
+
```

Comment:
Could we firm up the language here a bit perhaps? "MUST be able" seem to specify a capability when I feel it would be better to specify a behaviour.
```suggestion
Devices MUST reset ALL status transition counter properties in the following ways:

* On receiver activation when `autoResetStatusTransitionCounter` is enabled
* When a client invokes the `ResetStatusTransitionCounters` method

```